packages/locales/lib/human/de.json

```diff
@@ -734,5 +741,21 @@
   "reset_s2cells": "S2 Zellen zurücksetzen",
   "reset_spawnpoints": "Spawnpoints zurücksetzen",
   "reset_submission_cells": "Submission Zellen zurücksetzen",
-  "hisuian": "Hisui"
+  "hisuian": "Hisui",
+  "spacial_rend_range": "Raumschlag Effekt Bereich",
+  "key": "Schlüssel",
+  "ai": "KI",
+  "human": "Human",
+  "locales": "Lokalisierung",
+  "instructions": "Anleitung",
```

Comment:
is it singular or plural? Because in English it's plural, right?

Reply:
"instructions": "Anleitung", ?
singular -> look at the right top site